@here/harp-omv-datasource/lib/OmvData.ts

```diff
@@ -202,9 +202,9 @@ export class GeometryCommands {
                     const xx = geometry[cmdIndex++];
                     const yy = geometry[cmdIndex++];
 
-                    // tslint:disable:no-bitwise
-                    currX += (xx >> 1) ^ -(xx & 1);
-                    currY += (yy >> 1) ^ -(yy & 1);
+                    currX += xx % 2 ? (xx + 1) / -2 : xx / 2;
```

Comment:
Wonder what's the performance impact of that since we have now 3 divisions instead of 3 bit operations.
I guess we will not see any difference in our perftests.
Here is some related testcase that shows that modulo and divison is much slower:
https://jsperf.com/modulo-vs-bitwise/19